table-webview/webpack.config.js

```diff
@@ -0,0 +1,50 @@
+// @ts-check
+const path = require('path');
+
+const outputPath = path.resolve(__dirname, 'out');
+
+/**@type {import('webpack').Configuration}*/
+const config = {
+    target: 'web',
+
+    entry: path.resolve(__dirname, 'src/main.ts'),
+    output: {
+		filename: 'table-webview.js',
+        path: outputPath
+    },
+    devtool: 'eval-source-map',
+
+    resolve: {
+        extensions: ['.ts', '.tsx', '.js']
+    },
+    module: {
+        rules: [
+            {
+                test: /\.tsx?$/,
+                use: ['ts-loader']
+            },
+            {
+                test: /\.js$/,
+                use: ['source-map-loader'],
+                enforce: 'pre'
+            },
+            {
+                test: /\.css$/,
+                exclude: /\.useable\.css$/,
+                use: ['style-loader', 'css-loader']
+            },
+            {
+                test: /\.(ttf)$/,
+                loader: 'file-loader',
+                options: {
+                    name: '[name].[ext]',
+                    outputPath: '',
+                    publicPath: '..',
+                    postTransformPublicPath: (p) => `__webpack_public_path__ + ${p}`,
+                }
+            },
```

Comment:
This looks like not needed copypasta from another repository. Or do you intend to include own font files via .ttf?